### PR TITLE
Table names from cube. SPARKTA-184

### PR DESCRIPTION
**Pull request by sgomezg**
#### Description
Table names from cube. SPARKTA-184

#### Reviewer
@gserranojc    

#### Test
All OK
#### Coverage
No changes

#### Scalastyle
No info

### CODE REVIEW

File: sdk/src/main/scala/com/stratio/sparkta/sdk/AggregateOperations.scala [hunk 1]
```diff
@@ -50,63 +50,61 @@ object AggregateOperations {
...
+        if (timeDimension.isEmpty) None else Some(timeDimension))
 
+    val namesDim = dimensionValuesT.cube
     val (valuesDim, valuesAgg) = toSeq(dimensionValuesFiltered, aggregations ++ fixedAggregation)
 
+    val timeDimensionValue = filtered match {
+      case true => Seq(dimensionValuesT.time)
+      case false => Seq(DateOperations.millisToTimeStamp(dimensionValuesT.time))
+    }
+
```
Comment (gserranojc) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/AggregateOperations.scala]: If you rebase from master, the time.value calculation is new in aggregateOperations and SchemaFactory:

Mantain the new calculation:

Seq(getTimeFromDateType(dimensionValuesT.time, dateType))

**Comment by sgomezg** [comment]
Now is working. Rebase done, thanks @gserranojc

**Comment by danielcsant** [comment]
In the documentation it is mentioned where the cube name comes from. Please update documentation

**Comment by sgomezg** [comment]
Changes done in docs, thanks

**Comment by danielcsant** [comment]
+1

**Comment by gserranojc** [comment]
MILF

**Pull request merged**